### PR TITLE
APICLIENT-27: Add local news model and endpoint

### PR DESCRIPTION
**Pull request by amusameh**
This PR has been updated: 

1. LocalNews model, flow-types, and api has been updated to accommodate the retrieval of ids.
2. A new endpoint/API call has been created to retrieve a single LocalNews Item via id. 

Thought I'd let you know @klinzo since you were the one to approve the PR.

### CODE REVIEW

**Comment by steffenkleinle** [comment]
> It was written like this in the api so I used the same word

Please rename it to `timestamp` nevertheless, it's a typo. I'll forward it to the cms team. Thanks!

File: src/endpoints/createLocalNewsEndpoint.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import EndpointBuilder from '../EndpointBuilder'
+import type { JsonLocalNewsType } from '../types'
+import LocalNewsModel from '../models/LocalNewsModel'
+import moment from 'moment-timezone'
+import Endpoint from '../Endpoint'
+
+export const LOCALNEWS_ENDPOINT_NAME = 'news'
+
+type ParamsType = { city: string, language: string, channel: string }
```
Comment (steffenkleinle) [inline src/endpoints/createLocalNewsEndpoint.js]: AFAICS `channel` is unused. Either remove it or use it.

**Comment by steffenkleinle** [comment]
@svenseeberg will look into the `timestap` typo. Please let us know when it is fixed.

**Comment by steffenkleinle** [comment]
Should be fixed :)

**Review by steffenkleinle — APPROVED** [review]

File: src/types.js [hunk 2]
```diff
@@ -96,6 +96,14 @@ export type JsonEventType = {
...
 }
 
+export type JsonLocalNewsType = {
+  id: ?string,
+  timestamp: string,
+  title: string,
+  message: string
+}
+
+
```
Comment (mawadkittos) [inline src/types.js]: Please remove the line

File: src/endpoints/createLocalNewsEndpoint.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+export default (baseUrl: string): Endpoint<ParamsType, Array<LocalNewsModel>> =>
+  new EndpointBuilder(LOCALNEWS_ENDPOINT_NAME)
+    .withParamsToUrlMapper((params: ParamsType): string =>
+      `${baseUrl}/${params.city}/${params.language}/wp-json/extensions/v3/fcm?channel=news`
+    )
+    .withMapper((json: Array<JsonLocalNewsType>): Array<LocalNewsModel> => json
+      .map((localNews: JsonLocalNewsType) => {
+        return new LocalNewsModel({
+          id: localNews.id,
+          timestamp: moment.tz(localNews.timestamp, 'GMT'),
```
Comment (maxammann) [inline src/endpoints/createLocalNewsEndpoint.js:22]: @svenseeberg can we use here ISO8601 timestamps? That way we can potentially drop a library in our frontend.
@klinzo what do you think?

**Pull request closed without merging**